src/gui_impl/signal_browser/signal_graphics_item.cpp

```diff
@@ -72,6 +72,7 @@ SignalGraphicsItem::SignalGraphicsItem (QSharedPointer<SignalViewSettings const>
     connect(signal_view_settings.data(), SIGNAL(separatorEnabled(bool)), SLOT(enableSeparator(bool)));
     connect(signal_view_settings.data(), SIGNAL(gridColorChanged(QColor)), SLOT(updateGridColor(QColor)));
     connect(signal_view_settings.data(), SIGNAL(labelColorChanged(QColor)), SLOT(updateLabelColor(QColor)));
+    setToolTip("Ctrl+mouse wheel up/down to scale an indivicual channel");
```

Comment:
Typo: indivicual -> individual. Also, on a Mac the shortcut is Cmd + mouse wheel up/down - is there a way to dynamically adapt this tooltip?